river/feature_extraction/vectorize.py

```diff
@@ -156,6 +163,11 @@ def process_text(self, x):
             x = step(x)
         return x
 
+    def process_text_many(self, X: pd.Series):
+        for step in self.processing_steps:
+            X = X.apply(step)
+        return X
+
```

Comment:
Is this used? I see you're using `process_text` in a list comprehension underneath, which I think is better. I would only add a `process_text_many` if it brought performance benefits. The list comprehension makes it clear that this is "just" a for loop.